server/Makefile

```
@@ -0,0 +1,13 @@
+# Shortcuts for various dev tasks. Based on makefile from pydantic
+isort = isort src tests
+black = black src tests
+
+.PHONY: format
+format:
+	$(isort)
+	$(black)
+
+.PHONY: pep8
+pep8:
+	pip install -c requirements.txt pep8-naming flake8-black flake8-isort
```

Version constraints on the pep8 and flake8 packages are set via the `inmanta-dev-dependencies` package. This dependency is present in the `requirements.dev.txt` file and not in the `requirements.txt` file.
```suggestion
	pip install -c requirements.dev.txt pep8-naming flake8-black flake8-isort
```